css/style.css

```diff
@@ -248,31 +265,220 @@ body {
   color: #fff;
 }
 
+.stories {
+  height: 480px;
+}
+
+.stories__title {
+  text-align: center;
+  margin: 77px auto 41px auto;
+}
+
+.stories__items {
+  display: flex;
+  justify-content: space-between;
+}
+
+.story {
+  width: 307px;
+}
+
+.story__image {
+  width: 100%;
+  height: 148px;
+}
+
+.story__title {
+  font-size: 18px;
+  text-transform: none;
+  margin: 25px 0 10px 0;
+  letter-spacing: 0.02em;
+}
+
+.story__text {
+  margin: 0 0 10px 0;
+  font-size: 14px;
+  line-height: 17px;
+}
+
+.story__info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.story__button_text {
+  font-size: 12px;
+  color: #000;
+  font-weight: 800;
+}
+
+.story__button_WhiteArrow {
+ display: none;
+}
+
+.story__button_text:hover {
+  color: #aaa;
+}
+
+.story__date {
+  font-size: 12px;
+  opacity: .3;
+  font-weight: 700;
+  margin: 0;
+}
+
+.gallery {
+  height: 258px;
+}
+
+.gallery__title {
+    margin: 0 0 12px 0;
+    text-align: end;
+    font-size: 18px;
+    line-height: 17px;
+    text-transform: uppercase;
+    opacity: .3;
+    letter-spacing: .04em;
+}
+
+.gallery__photos {
+  display: flex;
+  width: 100%;
+}
+
+.gallery__photo {
+  width: 147.14px;
+  height: 147.14px;
+}
+
+.gallery__photo_tablet {
+    display: none;
+}
+
+.footer {
+  background: #000;
+  height: 297px;
+  color: #fff;
+}
+
+.footer__navigation {
+  display: flex;
+  justify-content: space-around;
+  padding: 0 147px 35px 156px;
+}
+
+.navigation__title {
+  font-size: 12px;
+  line-height: 11px;
+  margin: 66px 0 11px 0;
+  font-weight: 700;
+}
+
+.navigation__items {
+  display: flex;
+  flex-direction: column; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=6017130)